src/sidebars/favourites/favourite-preview.html

```diff
@@ -12,7 +13,10 @@ <h4 class="pull-left"><strong>{{stock.code | uppercase}}</strong></h4>
                 </div>
             </div>
             <div class="row">
-                <div class="col-xs-2 col-xs-offset-10">
+                <div class="col-xs-9" style="padding-left: 0px">
+                    <svg id="{{stock.code}}chart" width="100%" height="65px" ng-init="renderChart(stock)"></svg>
+                </div>
```

Comment:
I've read in the past that one should be wary of using ng-init (https://docs.angularjs.org/api/ng/directive/ngInit) -- this may well be a good use of it though. What do you think?